wiki_doc

```diff
@@ -0,0 +1,190 @@
+## Valkey Commands
+
+For information on the supported commands and their corresponding parameters, we recommend referring to the documentation in the code. This documentation provides in-depth insights into the usage and options available for each command.
+
+### Command String Arguments
+
+Valkey stores strings as sequences of bytes, so that may include text, serialized objects, or binary arrays. As such, to pass Valkey strings as arguments to commands, or receive Valkey strings in responses, GLIDE offers `GlideString`, which represents two APIs:
+
+1. `string`: an UTF-8 string.
+2. `Buffer`: an array of bytes.
+
+```typescript
```

Comment:
add link

Reply:
I have added the link to above `GlideString` so I am going to remove this.